CREATED CODE-OF-ETHICS PAGE

chiscookeke11 opened the pull request:
Created Code-of-ethics page

ONEONUORA commented:
Pls sync and fork with the master branch and git pull. This will be the last time, I will tell you this.

Import the navbar and footer in your code-of-ethics page .

ONEONUORA commented:
All failed

./src/app/code-of-ethics/page.tsx
1:8  Error: 'Image' is defined but never used.  @typescript-eslint/no-unused-vars

chiscookeke11 commented:
On it

On Fri, 24 Jan 2025, 19:31 Onuora Ebube Ebuka, ***@***.***>
wrote:

> All failed
>
> ./src/app/code-of-ethics/page.tsx
> 1:8 Error: 'Image' is defined but never used.
> @typescript-eslint/no-unused-vars
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/skill-mind/SkillNet-frontend/pull/35#issuecomment-2613148263>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BAYCM7AXJBZH5AWPWK2UYM32MKBGXAVCNFSM6AAAAABV2BSXL6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMMJTGE2DQMRWGM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

chiscookeke11 commented:
Please check review again
I just removed the line of code and my branch is in sync with the main

On Fri, 24 Jan 2025, 19:33 okeke emmanuel, ***@***.***> wrote:

> On it
>
> On Fri, 24 Jan 2025, 19:31 Onuora Ebube Ebuka, ***@***.***>
> wrote:
>
>> All failed
>>
>> ./src/app/code-of-ethics/page.tsx
>> 1:8 Error: 'Image' is defined but never used.
>> @typescript-eslint/no-unused-vars
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/skill-mind/SkillNet-frontend/pull/35#issuecomment-2613148263>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/BAYCM7AXJBZH5AWPWK2UYM32MKBGXAVCNFSM6AAAAABV2BSXL6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMMJTGE2DQMRWGM>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>

chiscookeke11 commented:
Please review
I've just implemented all the changes

On Fri, 24 Jan 2025, 19:48 okeke emmanuel, ***@***.***> wrote:

> Please check review again
> I just removed the line of code and my branch is in sync with the main
>
> On Fri, 24 Jan 2025, 19:33 okeke emmanuel, ***@***.***>
> wrote:
>
>> On it
>>
>> On Fri, 24 Jan 2025, 19:31 Onuora Ebube Ebuka, ***@***.***>
>> wrote:
>>
>>> All failed
>>>
>>> ./src/app/code-of-ethics/page.tsx
>>> 1:8 Error: 'Image' is defined but never used.
>>> @typescript-eslint/no-unused-vars
>>>
>>> —
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/skill-mind/SkillNet-frontend/pull/35#issuecomment-2613148263>,
>>> or unsubscribe
>>> <https://github.com/notifications/unsubscribe-auth/BAYCM7AXJBZH5AWPWK2UYM32MKBGXAVCNFSM6AAAAABV2BSXL6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMMJTGE2DQMRWGM>
>>> .
>>> You are receiving this because you authored the thread.Message ID:
>>> ***@***.***>
>>>
>>
